# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/gamepad/ipc/GamepadTestChannelChild.h
# Commit: ec6a64b639a8
# Full Hash: ec6a64b639a82e957376c09a71aef7ce62b3a5aa
# Author: Chris Martin <cmartin@mozilla.com>
# Date: 2020-12-03 09:47:26
# Regressor Bug: 1657404
# File Overlap Count: 3
# Description:
#   Bug 1657404 - Implement a strongly-typed, service-dependent gamepad handle r=handyman,aklotz
#   
#   Currently, the gamepad code uses a uint32_t as a handle and does some trickery
#   with it by trying to create a unique ID and adding an offset to it for VR code.
#   
# ==============================================================================

diff -r 6792f28b99bd -r ec6a64b639a8 dom/gamepad/ipc/GamepadTestChannelChild.h
--- a/dom/gamepad/ipc/GamepadTestChannelChild.h	Wed Dec 02 22:36:35 2020 +0000
+++ b/dom/gamepad/ipc/GamepadTestChannelChild.h	Wed Dec 02 23:06:05 2020 +0000
@@ -33,8 +33,8 @@
   explicit GamepadTestChannelChild(GamepadServiceTest* aGamepadServiceTest);
   ~GamepadTestChannelChild() = default;
 
-  mozilla::ipc::IPCResult RecvReplyGamepadIndex(const uint32_t& aID,
-                                                const uint32_t& aIndex);
+  mozilla::ipc::IPCResult RecvReplyGamepadHandle(const uint32_t& aID,
+                                                 const GamepadHandle& aHandle);
 
   WeakPtr<GamepadServiceTest> mGamepadServiceTest;
 